src/service/template-impl.js

```diff
@@ -69,6 +69,15 @@ export class BaseTemplate {
     // Subclasses may override.
   }
 
+  /**
+   * To be implemented by subclasses.
```

Comment:
"Bypasses template rendering and directly sets HTML. Should only be used for server-side rendering case. To be implemented by subclasses."